### PR TITLE
Changes workflows to run for the new release branch

### PR DESCRIPTION
**Pull request by saltiyazan**
# Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Checklist

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added tests that validate the behaviour of the software.
- [x] I validated that new and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules. --> merge blocked
- [ ] I have bumped the version of any required library.

### CODE REVIEW

File: .github/workflows/accessd.yml [hunk 1]
```diff
@@ -94,7 +94,7 @@ jobs:
           yes | juju kill-controller integration-tests-$GITHUB_RUN_ID
 
   orc8r-accessd-charmhub-upload:
-    if: github.ref_name == 'main'
+    if: github.ref_name == '1.6'
```
Comment (gruyaume) [inline .github/workflows/accessd.yml:30]: We probably won't want instances of those files for every magma release. We already have quite a bit of them.

Comment (saltiyazan) [reply .github/workflows/accessd.yml:30]: Could you please clarify which files you mean?

Comment (gruyaume) [reply .github/workflows/accessd.yml:30]: All of those touched by this PR

Comment (saltiyazan) [reply .github/workflows/accessd.yml:30]: Aren't they just the same files living in a different branch? why would we differ from main in our new release branches?
or should change this design even in main?

Comment (saltiyazan) [reply .github/workflows/accessd.yml:30]: I updated the PR, do you think this is a better design?

**Review by Gmerold — APPROVED** [review]

File: .github/workflows/accessd.yml (hunk 1 above)
Comment (saltiyazan) [reply .github/workflows/accessd.yml:30]: Based on the decision made by the team, to have these files release specific, I reverted to the original design.

**Review by Gmerold — APPROVED** [review]

**Review by ghislainbourgeois — APPROVED** [review]

File: .github/workflows/accessd.yml [hunk 2]
```diff
@@ -55,19 +59,19 @@ jobs:
...
           /usr/bin/sudo snap install jq
-          /usr/bin/sudo snap install charm --classic --channel=latest/stable
-          /usr/bin/sudo snap install charmcraft --classic --channel=latest/stable
+          /usr/bin/sudo snap install charm --classic --channel=$CHARM_CHANNEL
+          /usr/bin/sudo snap install charmcraft --classic --channel=$CHARMCRAFT_CHANNEL
           /usr/bin/sudo snap install juju-bundle --classic --channel=latest/stable
 
           echo "# Install microk8s"
-          /usr/bin/sudo snap install microk8s --classic
+          /usr/bin/sudo snap install microk8s --channel=$MICROK8S_CHANNEL --classic
```
Comment (Gmerold) [inline .github/workflows/accessd.yml]: Wait, `MICROK8S_CHANNEL` points to a strictly confines snap, so I guess we don't nees `--classic` anymore.

**Review by Gmerold — APPROVED** [review]

**Review by sanchezfdezjavier — APPROVED** [review]

**Pull request merged**